examples/SampleApp/src/icons/Pin.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { IconProps, RootPath, RootSvg } from '../utils/base';
```

Comment:
Is it intentional that this icon uses components from `../utils/base`, while icons/Message uses components from `react-native-svg` directly? Might be worth updating `viewBox` on `RootSvg` to use the `height` and `width` props rather than the hard coded values it has right now, as the defaulted values for height and width are the same as the hard coded values, maybe?

Reply:
I didn't get you on this thing, can you elaborate?

Reply:
@khushal87 How about we jump on a huddle some time during work tomorrow, and I can explain what I mean there? I can ping you when I'm at the office tomorrow morning, for example.